R/plot.R

```diff
@@ -181,11 +181,13 @@ combine_hp_performance <- function(trained_model_lst) {
 #' plot_hp_performance(hp_metrics$dat, lambda, AUC)
 #' }
 plot_hp_performance <- function(dat, param_col, metric_col) {
-  abort_packages_not_installed("ggplot2")
+  abort_packages_not_installed('ggplot2')
   return(dat %>%
-    ggplot2::ggplot(ggplot2::aes(
-      group = {{ param_col }},
-      y = {{ metric_col }}
-    )) +
-    ggplot2::geom_boxplot())
+           dplyr::group_by({{ param_col }}) %>% 
+           dplyr::summarise(mean_AUC = mean( {{metric_col}} ), sd_AUC = sd( {{metric_col}} )) %>% 
+           ggplot2::ggplot(ggplot2::aes(x = {{ param_col }},
+                                        y = mean_AUC)) +
```

Comment:
We can't assume AUC was the metric used. These can't be hard-coded.

Reply:
See example here: https://github.com/SchlossLab/tidy-eval/tree/key#4-bonus-write-a-function-that-names-a-new-column-based-on-the-variable